src/tests/Directory.Build.targets

```diff
@@ -161,7 +161,7 @@
 
      <!-- There are currently no native project binaries on wasm -->
      <Error  Text="The native project files are missing in $(NativeProjectOutputFolder) please run build from the root of the repo at least once"
-             Condition="'@(NativeProjectBinaries)' == '' And '$(TargetArchitecture)' != 'wasm'" />
+             Condition="'@(NativeProjectBinaries)' == '' And '$(TargetArchitecture)' != 'wasm' And '$(TargetOS)' != 'Android'"/>
```

Comment:
Could the comment above be updated to include Android if there are no project binaries for android either?

Reply:
Fixed.